lib/csv_row_model/model/columns.rb

```diff
@@ -49,6 +61,7 @@ def _columns
         # @option options [Hash] :type class you want to automatically parse to (by default does nothing, equivalent to String)
         # @option options [Hash] :parse a Proc for parsing the cell
         # @option options [Hash] :default default value of the column if it is blank?, can pass Proc
+        # @option options [Hash] :friendly_name human friendly string of the column name, by default column_name
```

Comment:
yep. maybe header is a better name for the options? https://github.com/FinalCAD/csv_row_model/issues/2